e2e/sdk.ERC1155.e2e.test.ts

```diff
@@ -296,7 +296,7 @@ describe('SDK - ERC1155 - contract interaction (deploy, load and mint)', () => {
     );
 
     const transferList3: any = await sdk.api.getTransfersByBlockNumber({
-      blockHashNumber: transfer[0].blockNumber,
+      blockNumber: transfer[0].blockNumber,
     });
     expect(
       transferList3.transfers.filter((tx: any) => tx.transactionHash === txHash.hash)[0],
```

Comment:
you could add here a check for your endpoint as well, which would be coping the lines from 298 to 303 and add them right after replacing getTransfersByBlockNumber with **getTransfersByBlockHash**